config/locales/en_LK.yml

```diff
@@ -52,28 +52,51 @@ en-LK:
   bs_below_200_copy:
     numerator: "Patients with RBS/PPBS <200, FBS<126, or Hba1c <7.0 at their last visit in the last 3 months"
     numerator_fbs: "Patients with FBS <126mg/dL, or HbA1c <7% at their last visit in the last 3 months"
+    numerator_dm: "Patients with RBS/PPBS <200, FBS<126 or Hba1c <7.0 at their last visit in the last 3 months"
+    numerator_dm_fbs: "Patients with FBS <126mg/dL or HbA1c <7% at their last visit in the last 3 months"
```

Comment:
These new keys have the exact text as the keys above them. Any particular reason for adding these new keys ?